src/lib_ccx/params.c

```diff
@@ -422,12 +422,12 @@ void print_usage(void)
 	mprint("\n");
 	mprint("In general, if you want English subtitles you don't need to use these options\n");
 	mprint("as they are broadcast in field 1, channel 1. If you want the second language\n");
-	mprint("(usually Spanish) you may need to try -2, or -cc2, or both.\n\n");
+	mprint("(usually Spanish) you may need to try --2, or --CC2, or both.\n\n");
 	mprint("Input formats:\n");
 	mprint("       With the exception of McPoodle's raw format, which is just the closed\n");
 	mprint("       caption data with no other info, CCExtractor can usually detect the\n");
 	mprint("       input format correctly. To force a specific format:\n\n");
-	mprint("                  -in=format\n\n");
+	mprint("                 --inp [format]\n\n");
```

Comment:
Why inp instead of in?

Reply:
in is a keyword in [dart](https://dart.dev/guides/language/language-tour#keywords) So I can't keep the name in inside my SettingsModel in my GUI. I could handle the --in case manually on the GUI side but thought it would be easier here. Should I revert this?

Reply:
Yes. No way we're going to change the C or Rust code because a totally unrelated program, in a different language, has a keyword. 

I'd say your SettingModels is trying to do directly what clearly needs a translation table...

Reply:
Ok I have reverted this back to `--in format`